# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/moz.build
# Commit: 2be4a1d5ac07
# Full Hash: 2be4a1d5ac0724aab233faaf469c9b8f5e1d2454
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2022-08-17 15:40:28
# Regressor Bug: 1740530
# File Overlap Count: 1
# Description:
#   Bug 1740530 - patch 1 - Move COLR font support functions from gfxUtils into their own class. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152036
# ==============================================================================

diff -r 1e2092c319eb -r 2be4a1d5ac07 gfx/thebes/moz.build
--- a/gfx/thebes/moz.build	Wed Aug 17 10:24:52 2022 +0000
+++ b/gfx/thebes/moz.build	Wed Aug 17 10:59:33 2022 +0000
@@ -17,6 +17,7 @@
 XPIDL_MODULE = "gfx"
 
 EXPORTS += [
+    "COLRFonts.h",
     "DrawMode.h",
     "gfx2DGlue.h",
     "gfxAlphaRecovery.h",
@@ -195,6 +196,7 @@
 
 UNIFIED_SOURCES += [
     "CJKCompatSVS.cpp",
+    "COLRFonts.cpp",
     "gfxAlphaRecovery.cpp",
     "gfxBaseSharedMemorySurface.cpp",
     "gfxBlur.cpp",
